.infrastructure/helm-chart/todoapp/charts/mysql/templates/statefulSet.yml

```diff
@@ -0,0 +1,91 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: {{ .Chart.Name }}-stateful-set
+  namespace: {{ .Values.common.namespace }}
+spec:
+  selector:
+    matchLabels:
+      app: {{ .Values.mysql.labelName }}
+  replicas: {{ .Values.statefulSet.replicas }}
+  serviceName: {{ .Chart.Name }}-service
+  template:
+    metadata:
+      labels:
+        app: {{ .Values.mysql.labelName }}
+    spec:
+      containers:
+      - name: {{ .Chart.Name }}-container
+        image: {{ .Values.mysql.image.name }}:{{ .Values.mysql.image.tag }}
```

Comment:
Ensure that `.Values.mysql.image.name` and `.Values.mysql.image.tag` are defined in your `values.yaml` file to specify the correct image for the container.